src/pug/blog/report_28.pug

```diff
@@ -0,0 +1,32 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #28
+
+block content
+  h2 Progress report #28
+  +news_header('18 April 2025', 'report_27', '')
+
+  h3 Alpakka 1.0 pre-orders being delivered
+  p On 12th of March 2025 we started shipping for Alpakka 1.0 DIY kits, as for today #[em 75% of the pre-orders] has been fulfilled.
+  p From the logistics point of view, we have #[em 2 different queues], depending if the order has 3d-printed parts (and goes into a box) or it does not (and goes into a padded envelope). The envelopes are going out faster, while with the 3d-printed parts the production is constrained by the speed of the printers.
+  p Overall, we are #[em happy with the feedback received] from the first Alpakka 1.0 owners, and we hope catch up with all pre-orders in a few more weeks.
+
+  hr
+  h3 Press F to print respects
+  p To make things even more interesting, one of our printers, the Prusa Mini+ (#[em OG Input Labs printer]), had it first critical failure after more than #[em 5000 hours] of printing. 🫡
+  p The hotend PTFE tube was ruptured in the bottom part, and now we are awaiting for replacement parts. Hopefully the printer will get into its former glory soon, but meanwhile we will have reduced printing capabilities for at least a week.
+
+  hr
+  h3 Alpakka 0.97 kits discontinued on the shop
+  p The old Alpakka controller (aka #[em v0]) are no longer available in the Input Labs shop. But we will keep the #[em electronics kit] in stock for a while, in case someone needs to replace aged components at some point.
+  p Just to clarify, the new 1.0 Alpakka fully replaces the previous Alpakka design, and can also be used wired (without dongle nor battery).
```

Comment:
- rephrase the "wired" part. Something like "in wired and wireless mode", or so.
- This might be a good place to mention / re-iterate that the Marmota replaces the pico and is needed in addition to the PCB (and the electronics kit, in the case of the partially soldered version)